frontend/packages/design-system/src/themes/Text.ts

```diff
@@ -0,0 +1,51 @@
+import fontConfig from '../tokens/fontConfig';
+
+export default {
+  defaultProps: {
+    letterSpacing: '0em',
+    textAlign: 'auto',
+  },
+  variants: {
+    display: {
+      fontFamily: fontConfig.Roboto['700'].normal,
+      fontSize: '44px',
```

Comment:
In `fontConfig` I would typically have a `fontSize` key with the font sizes. IBM use rem units for their's which I quite like. Even though we can't use rem for react native we could set a base font size `1` (typically 16px) and define each font size from that, eg:
```
fontSizes: {
 '0.875': '14px',
  '1': '16px',
  '1.125': '18px',
}
```
we could event compute this:
```
const baseFontSize = 16;
const scales = [0.875, 1, 1.125];
const fontSizes = scales.reduce((acc, cur) => ({ ...acc, [cur]: `${cur * baseFontSize}px` }), {});
```

Perhaps this is overkill and while the typography is still being worked on we can leave it hard coded and discuss this with the designers.
https://www.carbondesignsystem.com/guidelines/typography/overview/

Reply:
I'm not sure how useful this is if the base font size is not configurable. also not sure about the 0.875 keys, native base only seems to work with xs, sm and such